roles/common/defaults/main.yml

```diff
@@ -10,9 +10,9 @@ ats_home: "{{ binaries_folder }}/transform-service"
 
 # Due to https://github.com/ansible/ansible/issues/64957
 # Using default() filter for hostvars 1st level key requires ansible 2.10+
-nginx_host: "{{ groups['nginx'] | map('extract', hostvars, ['ansible_default_ipv4','address']) | first | default('127.0.0.1') }}"
+nginx_host: "{{ groups['nginx'] | map('extract', hostvars, ['ansible_default_ipv4','address']) | first | default('127.0.0.1') }}"
```

Comment:
those are my bad! I really ned to understand why that happen to me!

Reply:
non-printable characters, who knows xD